lib/models/DeveloperOAuth.js

```diff
@@ -150,6 +150,14 @@ class DeveloperOAuth {
         currentToken = (req.session.accessTokens || []).find(
           (token) => token.merchantId === currentMerchantId && token.performerId === currentPerformerId,
         );
+
+        const prevSession = req.session;
+        const currentSession = (currentStaff || {}).public_session_id_hash;
```

Comment:
please name the variable more accurately.
* it is a session_hash (prevSession is a session, currentSession is just a string)
* it is sso session